src/thunderbots/software/ai/navigator/obstacle/obstacle.cpp

```diff
@@ -0,0 +1,117 @@
+
+#include "obstacle.h"
+
+const Polygon& Obstacle::getBoundaryPolygon() const
+{
+    return _polygon;
+}
+
+Obstacle::Obstacle() : _polygon{Point(0, 0), Point(0, 1), Point(1, 1), Point(1, 0)} {}
+
+Obstacle::Obstacle(Polygon polygon) : _polygon(polygon) {}
+
+Obstacle Obstacle::createPlaceholderObstacle()
+{
+    return Obstacle();
+}
+
+Obstacle Obstacle::createRobotObstacle(const Robot& robot, bool enable_velocity_cushion)
+{
+    double tick_length = 1.0;
```

Comment:
I'm a bit confused about what we're doing here. In general, we want to scale with respect to time in seconds, not in ticks?

Reply:
I will clarify that it's tick_length_in_seconds